.github/workflows/publish-snapshot.yml

```diff
@@ -0,0 +1,35 @@
+name: Publish Snapshot
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  maven:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-java@v3
+        with:
+          distribution: temurin
+          java-version: 11
+          server-id: ossrh
+          server-username: MAVEN_USERNAME
+          server-password: MAVEN_PASSWORD
+      - name: Publish snapshot
+        run: mvn --batch-mode -no-transfer-progress --update-snapshots deploy
+        env:
+          MAVEN_USERNAME: ${{ vars.OSSRH_USERNAME }}
+          MAVEN_PASSWORD: ${{ secrets.OSSRH_TOKEN }}
+      - name: Set up Docker Buildx
+        uses: docker/setup-buildx-action@v2
+      - name: Login to Docker Hub
+        uses: docker/login-action@v2
+         with:
+           username: ${{ vars.DOCKER_USERNAME }}
+           password: ${{ secrets.DOCKER_PASSWORD }}
+      - name: Build and push
+        uses: docker/build-push-action@v4
+         with:
+           push: true
+           tags: user/app:latest
```

Comment:
The `tags` should be `cqframework/cqf-ruler` and `alphora/cqf-ruler` (for backwards compatibility), right?

Also, there's a `build-arg` for setting the container with a git hash. See here:

https://github.com/cqframework/cqf-ruler/blob/master/Dockerfile#L3